File: src/test/java/com/datastruct/arraystack/scala/ArrayStackTest.scala
package com.datastruct.arraystack.scala

object ArrayStackTest {
  def main(args: Array[String]): Unit = {
    val arrayStack = ArrayStack[Int](2)

    println(arrayStack.isEmpty)
    println(arrayStack.getSize)

    arrayStack.push(1)
    arrayStack.push(2)
    arrayStack.push(3)

    println(arrayStack.isEmpty)
    println(arrayStack.getSize)


  }
}

File: src/test/java/com/datastruct/arraystack/java/ArrayStackTest.java
package com.datastruct.arraystack.java;

public class ArrayStackTest {
    public static void main(String[] args) {
        ArrayStack<Integer> arrayStack = new ArrayStack<>(10);

        arrayStack.push(1);

        System.out.println(arrayStack.getSize());
        System.out.println(arrayStack.isEmpty());
        System.out.println(arrayStack.top());
        System.out.println(arrayStack.pop());
        System.out.println(arrayStack.getSize());
        System.out.println(arrayStack.isEmpty());
    }
}
